tools/parity_scale_decoder/src/generate_map.rs

```diff
@@ -26,7 +26,7 @@ impl<T> TypeName for T {
     }
 }
 
-/// Newtype which has `type_name()` method when `T` implements [`IntoSchema`]
+/// New type which has `type_name()` method when `T` implements [`IntoSchema`]
```

Comment:
This is a technical term. Please leave as is.

Reply:
ok!